python-sdk/tests/benchmark/config.json

```diff
@@ -20,56 +20,1242 @@
           "database": "postgres"
         }
       }
+    },
```

Comment:
Is this file generated using a script or manually?

Reply:
Manually. Why?